contributors/devel/development.md

```diff
@@ -247,6 +247,8 @@ _If you have upstream write access_, please refrain from using the GitHub UI for
 creating PRs, because GitHub will create the PR branch inside the main
 repository rather than inside your fork.
 
+When pushing commits to a pull request, don't force push. Force pushing can corrupt your pull request.
```

Comment:
@gyliu513  We should use force push before creating a PR since the commits were pushed to our own repo. When updating a PR with additional commits, it is possible to corrupt the PR if using force pushing according to Github docs.